runtime/queries/dart/locals.scm

```diff
@@ -0,0 +1,8 @@
+(class_definition
```

Comment:
For locals, take a look at the golang definition: https://github.com/helix-editor/helix/blob/master/runtime/queries/go/locals.scm

`@local.scope` to tag scopes, `@local.definition` to tag variable definitions and `@local.reference` for references

Reply:
Sorry, I should have looked before.